bindings/haskell/Cargo.toml

```diff
@@ -34,4 +34,4 @@ doc = false
 [dependencies]
 chrono = "0.4"
 log = { version = "0.4", features = ["std"] }
-opendal = { path = "../../core" }
+opendal = { version = "0.44.2", path = "../../core" }
```

Comment:
Could you provide a link to show which action pipeline failed?

Reply:
https://github.com/apache/opendal/issues/4094